packages/chart/src/Pie.ts

```diff
@@ -52,10 +54,18 @@ export class Pie extends SVGWidget {
     }
 
     calcOuterRadius() {
-        const maxTextWidth = this.textSize(this.data().map(d => d[0]), "Verdana", 12).width;
+        const maxTextWidth = this.textSize(this.data().map(d => this.getLabelText({ data: d })), "Verdana", 12).width;
         return Math.min(this._size.width - maxTextWidth * 2 - 20, this._size.height - 12 * 3) / 2 - 2;
     }
-
+    calcTotalValue(): number {
+        return this.d3Pie(this.data()).reduce((acc, d) => {
+            return acc + d.data[1];
+        }, 0);
+    }
+    getLabelText(d) {
+        const _perc = ((d.data[1] / this._totalValue) * 100).toFixed(1).split(".0").join("");
+        return d.data[0] + (this.showSeriesPercentage() ? `: ${_perc}%` : "");
```

Comment:
Minor:  no need to calc percentage if its not needed.
Style comment:  For consistency, don't add an "_" to local variables,